### PR TITLE
modified  bug_report_template.yaml

### LINKED ISSUE #5960: Update `System information` section and add `MLflow version` section in `bug_report_template.yaml`
We want to update [`bug_report_template.yaml`](https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/bug_report_template.yaml) to make it match [`ui_bug_report_template.yaml`](https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/ui_bug_report_template.yaml):

### bug_report_template.yaml

image

### ui_bug_report_template.yaml

image

`bug_report_template.yaml` needs to be updated as follows:

```diff
# git diff output

diff --git a/.github/ISSUE_TEMPLATE/bug_report_template.yaml b/.github/ISSUE_TEMPLATE/bug_report_template.yaml
index c6b55dcad..16d9cbfee 100644
--- a/.github/ISSUE_TEMPLATE/bug_report_template.yaml
+++ b/.github/ISSUE_TEMPLATE/bug_report_template.yaml
@@ -20,16 +20,20 @@ body:
         - No. I cannot contribute a bug fix at this time.
     validations:
       required: true
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: MLflow version
+      description: MLflow version (run `mlflow --version`) or commit SHA if you have MLflow installed from source (run `pip freeze | grep mlflow`).
+
   - type: textarea
     attributes:
       label: System information
       description: |
         Describe the system where you encountered the bug.
       value: |
-        - **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
         - **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
-        - **MLflow installed from (source or binary)**:
-        - **MLflow version (run ``mlflow --version``)**:
         - **Python version**:
         - **yarn version, if running the dev UI**:
     validations:
```

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
## Related Issues/PRs

Close #5960
 
## What changes are proposed in this pull request?
Update System information section and add MLflow version section in bug_report_template.yaml 
(Please fill in changes proposed in this fix)

## How is this patch tested?

(Details)

## Does this PR change the documentation?

- [ 

- [x] ] No. You can skip the rest of this section.

1. Check the status of the `ci/circleci: build_doc` check. If it's successful, proceed to the
   next step, otherwise fix it.
2. Click `Details` on the right to open the job page of CircleCI.
3. Click the `Artifacts` tab.
4. Click `docs/build/html/index.html`.
5. Find the changed pages / sections and make sure they render correctly.

### Is this a user-facing change?

- [ ] No. You can skip the rest of this section.
- [x] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [ ] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: MLflow Model server, model deployment tools, Spark UDFs
- [ ] `area/server-infra`: MLflow Tracking server backend
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface
- [ ] `area/uiux`: Front-end, user experience, plotting, JavaScript, JavaScript dev server
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by harupy** [comment]
@SaumyaBhushan Thanks for the PR, can you take a look at #5960 and where the green lines are added?

**Comment by harupy** [comment]
You can check whether the form renders correctly in https://github.com/SaumyaBhushan/mlflow/blob/Issue%235960/.github/ISSUE_TEMPLATE/bug_report_template.yaml.

**Comment by SaumyaBhushan** [comment]
@harupy I think it is OK now.

File: .github/ISSUE_TEMPLATE/bug_report_template.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ labels: 'bug'
 title: '[BUG]'
 
 body:
+
```
Comment (harupy) [inline .github/ISSUE_TEMPLATE/bug_report_template.yaml:7]:
```suggestion
```

Comment (harupy) [reply .github/ISSUE_TEMPLATE/bug_report_template.yaml:7]: @SaumyaBhushan Can you remove this line?

**Comment by SaumyaBhushan** [comment]
Hey @harupy, Is it ok now?

**Comment by harupy** [comment]
Looks good! Can you fix the DCO check failure?

**Review by harupy — APPROVED** [review]

**Pull request merged**